css-positioning/absolute.css

```diff
@@ -0,0 +1,48 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 20px;
+  left: 250px;
+  width: 300px;
```

Comment:
Does your output still look correct with this? I would think it would mess up the layout 🤔